js/state/siteUtil.js

```diff
@@ -109,9 +109,6 @@ module.exports.addSite = function (sites, siteDetail, tag, originalSiteDetail) {
     tags = tags.toSet().add(tag).toList()
   }
 
-  // We don't want bookmarks and other site info being renamed on users if they already exist
-  // The name should remain the same while it is bookmarked forever.
-  const customTitle = typeof siteDetail.get('customTitle') === 'string' ? siteDetail.get('customTitle') : (siteDetail.get('customTitle') || oldSite && oldSite.get('customTitle'))
```

Comment:
did you check to make sure this isn't regressed? Use git blame on the task to see the original bug.